pkg/cloudprovider/rhsm/subscription_manager.go

```diff
@@ -69,40 +64,40 @@ func NewRedHatSubscriptionManager(offlineToken string) (RedHatSubscriptionManage
 	if offlineToken == "" {
 		return nil, errors.New("RedHatSubscriptionManager offline token cannot be empty")
 	}
-
 	return &defaultRedHatSubscriptionManager{
-		client: &http.Client{
-			Timeout: 10 * time.Second,
-		},
-		authURL:         "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token",
-		apiURL:          "https://api.access.redhat.com/management/v1/systems",
-		offlineToken:    offlineToken,
-		requestsLimiter: 100,
```

Comment:
Please revert back this line as this break the current logic and run the rhsm into an infinite loop which is why the e2e test will fail probably

Reply:
ok, I removed this by mistake